backend/db-setup/generate_db_data.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+
+import sys
+import csv
+import argparse
+import random
+import faker
+import importlib
+
+def address(result):
+    return [[oid, result['faker'].unique.address().replace('\n', ', ')]
+            for oid in range(result['lines'])]
+
+def city(result):
+    return [[oid, result['faker'].unique.city()] for oid in range(result['lines'])]
+
+
+def company(result):
+    return [[oid, result['faker'].unique.company()] for oid in range(result['lines'])]
+
+def email(result):
+    return [[oid, result['faker'].unique.email()] for oid in range(result['lines'])]
+
+def lorem_ipsum(result):
+    return [[oid, result['faker'].unique.paragraph(nb_sentences=8)] for oid in range(result['lines'])]
+
+
+def postcode(result):
+    return [[oid, result['faker'].unique.postcode()] for oid in range(result['lines'])]
+
+def main(args):
+    generator_methods = [
+        'street_name', 'address', 'city', 'company', 'country', 'email', 
+        'first_name', 'iban', 'last_name', 'lorem_ipsum', 'postcode', 'siret'
```

Comment:
some of these values don't have generator methods defined
e.g. `generate_db_data.py --table street_name` is going to fail

Reply:
Thanks for pointing this out, should be all resolved now.